.jshintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Thanks for adding this, I agree that this repo should have hinting + linting but that should be part of a separate PR. (I'd also prefer eslint)